lib/node/urlStorage.js

```diff
@@ -140,15 +159,15 @@ export class FileUrlStorage {
     readFile(this.path, 'utf8', (err, data) => {
       if (err) {
         // return empty data if file does not exist
-        err.code === 'ENOENT' ? cb(null, {}) : cb(err)
+        if (err.code === 'ENOENT') cb(null, {})
+        else cb(err)
       } else {
         try {
-          data = !data.trim().length ? {} : JSON.parse(data)
+          const data2 = !data.trim().length ? {} : JSON.parse(data)
```

Comment:
I assume there is a rule disallowing the reassignment of arguments. Could we loosen up this rule? I actually think the previous code where reassignment is allowed is more reasonable. Or is there a specific reason against it?